src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -512,27 +531,41 @@ public Response deletePolicies(List<UUID> uuids) {
         // TODO Temp counter used to investigate the engine instability, remove ASAP.
         registry.counter("policies_ui_deletePolicies", new Tag("account", user.getAccount())).inc();
 
+        // TODO POL-649 Do we really need to return this?
```

Comment:
Yes, as the operation was not a single transaction, some policies could fail to be deleted. The UI needs to know which was deleted.

Reply:
TODO removed.